bliss/sleep.py

```diff
@@ -223,6 +225,7 @@ def tile_images_map_estimate(self, images):
                 max_detections,
                 n_galaxy_params,
             )
+            # NOTE: tile_est["locs"].shape = (2, 100, 1, 8)
```

Comment:
@ismael-mendoza  same as above. Can you recall what its second dimension 100 represents here?

Reply:
You may not hear from Ismael while he's traveling, but don't let that stop you. It's likely that 100 is the number of tiles in a 10x10 grid of tiles.